common/src/Utilities/Windows/WinMisc.cpp

```diff
@@ -108,7 +103,7 @@ wxString GetOSVersionString()
     else
         GetSystemInfo(&si);
 
-    if (VER_PLATFORM_WIN32_NT != osvi.dwPlatformId || osvi.dwMajorVersion <= 5)
+    if (VER_PLATFORM_WIN32_NT != osvi.dwPlatformId || osvi.dwMajorVersion <= 6 && osvi.dwMinorVersion <= 1)
```

Comment:
Maybe include Windows 8 as an unsupported operating system as well since that's EOL (I'm not referring to Windows 8.1, that's not EOL).

Reply:
Windows 8 should still work though, we're only flagging Win7 because we're removing something it needs. We don't want to cripple an OS just for lulz

Reply:
I agree, w8 as minimum is fine. Might want to update github readme too.

Reply:
Changing this number doesn't actually prevent things from running. It'll just show up in the logs differently.

It'll still run until something that is necessary for it to work gets removed. Windows 7 probably still works at this point, it's just that XAudio2 can't be used on it anymore.

I think if the OS is EOL then we should opt to not provide support for it even if it still works and encourage the user to upgrade.

Reply:
Alright then, CK go ahead with the change.

Reply:
I'm not sure if this if statement will work in its current form, what if the major/minor version is 5.2 (windows xp) ? it won't meet both conditions.  You might need to split it out so the minor version is checked afterwards

Reply:
if (VER_PLATFORM_WIN32_NT != osvi.dwPlatformId || osvi.dwMajorVersion <= 5 || (osvi.dwMajorVersion == 6 && osvi.dwMinorVersion <= 1))

That would work